thrall/app/lib/ThrallMessageConsumer.scala

```diff
@@ -43,7 +43,11 @@ object ThrallMessageConsumer extends MessageConsumer(
     withImageId(exports)(id => ElasticSearch.deleteImageExports(id))
 
   def updateImageUserMetadata(metadata: JsValue): Future[UpdateResponse] =
-    withImageId(metadata)(id => ElasticSearch.applyImageMetadataOverride(id, metadata \ "data"))
+    withImageId(metadata)(
+      id => ElasticSearch.applyImageMetadataOverride(
+        id, metadata \ "data", metadata \ "userMetadataLastModified"
+      )
+    )
```

Comment:
Since you asked us to be anal about styling, one way to avoid multi-line statements is to use temporary variables, e.g.

``` scala
  def updateImageUserMetadata(metadata: JsValue): Future[UpdateResponse] = {
    val data = metadata \ "data"
    val lastModified = metadata \ "userMetadataLastModified"
    withImageId(metadata)(id => ElasticSearch.applyImageMetadataOverride(id, data, lastModified))
  }
```

Bit academic here, and the last line is probably still a bit long, but you get the idea.